### DISCUSSION TITLE
Does this plugin not have full feature support for large files?

### DISCUSSION THREAD

**Discussion by RyanZurrin** [izhangzhihao/intellij-rainbow-brackets#2604] [Q&A] [answered]
I have some files that are over 1000 lines. Though I have enabled large file support, it seems like many of the features do not work on larger files, such as being able to hold ctrl + Right Click and have a whole block highlighted:
![image](https://user-images.githubusercontent.com/43255897/224502978-69213152-665b-41c4-9867-08b353f8fb8f.png)

When you click a bracket, it will also not draw the darker line color along the whole edge of the block as it does in shorter files:
![image](https://user-images.githubusercontent.com/43255897/224503038-71670cc4-b021-4329-b318-d2b4f0b1edf9.png)

Currently, on larger files, when trying to do similar, I am only getting the following:
![image](https://user-images.githubusercontent.com/43255897/224503101-17ac356f-56e2-4182-9b4e-023bab59dd7b.png)

Is this normal behavior?

**Comment by izhangzhihao** [comment, marked as answer]
Just uncheck this option or increase the number of lines:

<img width="1065" alt="image" src="https://user-images.githubusercontent.com/12044174/224517403-2c5aef9a-7946-4fcf-9312-fc2109d6aef3.png">

**Reply by RyanZurrin** [reply]
AWESOME!!! I figured I had to be missing something someplace. That is wonderful; thanks so much for the help. I enjoy the features of this plugin and want to say thanks for making it and for all the great work you put into this project!!!!
